src/bundle/Resources/public/js/scripts/core/date.time.picker.js

```diff
@@ -85,13 +84,17 @@ class DateTimePicker {
         if (typeof value === 'number' && value >= 0) {
             const flatpickrDate = this.flatpickrInstance.selectedDates[0];
 
+            if (flatpickrDate === undefined) {
+                return;
+            }
+
             if (isMinute) {
                 flatpickrDate.setMinutes(value);
             } else {
                 flatpickrDate.setHours(value);
             }
 
-            if (this.flatpickrConfig.minDate.getTime() > flatpickrDate.getTime()) {
```

Comment:
The JS error caused by this place is fixed in https://github.com/ibexa/admin-ui/pull/1303